pybossa/importers/csv.py

```diff
@@ -58,91 +58,179 @@ def error():
 
     return value
 
-class BulkTaskCSVImport(BulkTaskImport):
+class ReservedFieldProcessor(object):
+    def __init__(self, header):
+        self.header = header
+
+    reserved_fields = set([
+        'state',
+        'quorum',
+        'calibration',
+        'priority_0',
+        'n_answers',
+        'user_pref',
+        'expiration'
+    ])
+    is_input = False
+
+    @staticmethod
+    def can_process(header):
```

Comment:
The way this function is used is more as a factory method.  More like `create_or_none`.  No big deal.

Reply:
Changed the name to `create_if_can_process`.